test/test_property.py

```diff
@@ -142,6 +142,11 @@ def test_text_property_setting_list(self):
         tp.value = expected
         self.assertEqual(tp.value, [])
 
+    def test_owned_object_singleton(self):
+        cd = sbol.ComponentDefinition('cd')
+        cd.sequence = sbol.Sequence('seq')
```

Comment:
I think Sequence is a bad example here because it is deprecated in favor of `sequences`. Is it easy to choose another example?

Reply:
Yes, I made this test case more abstract using a generic `OwnedObject` annotation property